style.css

```diff
@@ -42,6 +42,61 @@ h2 {
   color: #6070ff;
 }
 
+img {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+button:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+button:disabled {
+  border: 1px solid #c1c7d0;
+  background-color: #fff;
+  color: #5e6c84;
+}
+
+button:active {
+  background: #2230d2;
+  color: #fff;
+}
+
+button:enabled {
+  border: 1px solid #7f8cff;
+  background: #fff;
+  color: #6070ff;
+}
+
```

Comment:
- [ ] Great job adding all this button styling. It is currently not rendering though. There is a convention for writing the pseudoclasses. Try putting enabled at the top because it is overriding other states.